concourse/pipelines/container-build.jsonnet

```
@@ -57,9 +62,14 @@ local buildcontainerimgjob = {
         ] +
         job.extra_steps +
         [
+          {
+            load_var: job.image + '-commit-sha',
+            file: job.input + '/.git/ref',
```

for anything other than simple append, let's use formatting:
```suggestion
            file: '%s/.git/ref' % job.input,
```

---

done!